test/verify/check-shell-menu

```diff
@@ -82,6 +85,7 @@ class TestMenu(testlib.MachineCase):
         # Clicking inside the iframed pages should close the docs menu
         b.click("#toggle-docs")
         b.wait_visible("#toggle-docs-menu")
+        b.assert_pixels("#toggle-docs-menu", "shell-docs-menu")
```

Comment:
This only has a single screen shot, I take it it doesn't look differently in the various sizes? Doing a dark theme snapshot would be good, though!

Reply:
This has multiple screenshots, the other pixel test asserts in one layout because it the test itself switches layout. We could indeed test test in a dark theme and a different layout it might be a bit invasive for the currently used test.